mapper/cumulocity/c8y_translator_lib/src/serializer.rs

```diff
@@ -55,6 +59,12 @@ impl C8yJsonSerializer {
             return Err(MeasurementStreamError::UnexpectedEndOfData.into());
         }
 
+        if !self.timestamp_present {
+            self.timestamp(self.default_timestamp)?;
```

Comment:
I feel the parser should add the timestamp, @Didier can clarify this.

Reply:
@PradeepKiruvale Hi, I think you meant @didier-wenzek.

Reply:
Hmm, exactly, sorry.